superset-frontend/cypress-base/cypress/e2e/dashboard/editmode.test.ts

```diff
@@ -771,7 +771,7 @@ describe('Dashboard edit', () => {
       visitEdit();
     });
 
-    it('should save', () => {
+    it.skip('should save', () => {
```

Comment:
@eschutho I'm not sure the fate of these tests... should we scrap them, or try to rekindle them?